catalog.tests.bom

```diff
@@ -87,7 +87,7 @@ brooklyn.catalog:
           - equals: true
         - type: org.apache.brooklyn.test.framework.SimpleShellCommandTest
```

Comment:
+1 A lot of this has been done in https://github.com/brooklyncentral/common-catalog-utils/blob/master/common-tests/src/main/resources/commontests/common.tests.bom#L171